index.js

```diff
@@ -100,8 +100,8 @@ cache.setup(() => {
   // Get an xapp token
   artsyXapp.init({ url: API_URL, id: CLIENT_ID, secret: CLIENT_SECRET }, () => {
     // Start the server
-    app.listen(PORT, () => {
-      console.log(`Force listening on port ${PORT}`)
-    })
+    if (module === require.main) {
```

Comment:
This tells the root Force app to not start listening unless we run the module directly (e.g. in `yarn start`), giving us more control when we set up end to end tests.